js/event.js

```diff
@@ -1,3 +1,3 @@
 function submit() {
-    alert("fuck you");
```

Comment:
why did you delete this?

Reply:
apologies, thought we should give the user a clearer message